packages/editor/src/components/projects/ProjectsPage.tsx

```diff
@@ -424,37 +422,19 @@ const ProjectPage = ({ studioPath }: { studioPath: string }) => {
       </ContextMenu>
 
       {installedProjects.length > 0 && (
-        <Accordion
-          title={`${t('editor.projects.installed')} (${installedProjects.length})`}
-          expandIcon={<MdArrowRight className="text-2xl" />}
-          shrinkIcon={<MdArrowDropDown className="text-2xl" />}
-          className="mb-3 mt-5 w-3/4"
-          open={true}
-        >
+        <Accordion title={`${t('editor.projects.installed')} (${installedProjects.length})`} open={true}>
```

Comment:
lets edit the translation string such that it can take the variable natively
for example https://github.com/ir-engine/ir-engine/blob/9bedb608eb5acaefe6a6e1836136925175ee36fe/packages/client-core/i18n/en/editor.json/#L33

Reply:
Okay. Lets keep it then